recipe/meta.yaml

```diff
@@ -149,10 +149,10 @@ outputs:
         - idds-client >=2.0.9
         - idds-doma >=2.0.9
         - iminuit >=2.24.0
-        - lmfit >=1.2.2
+        - lmfit >=1.2.2,<1.3
         - lsstdesc.coord >=1.3.0
         - lsst-ts-xml >=20.1.0
-        - matplotlib-base >=3.8.2
+        - matplotlib-base >=3.8.2,<3.9
```

Comment:
Why is this needed? (Skyproj at least is compatible)

Reply:
I took it from the `main` branch; I think it was for "possible API breakage" reasons.  I can try 3.9(.1), but I doubt we have good test coverage.